baekjoon_5/4344.py

```diff
@@ -0,0 +1,21 @@
+# 4344 - 평균은 넘겠지...
+
+import statistics
+
+num = int(input())
+
+for i in range(num):
+
+    score = list(map(int, input().split()))  # 첫번째에는 학생 수가 들어오므로 슬라이싱을 통해 1번째 항목부터 끝까지 평균을 구하면 될 듯하다.
+
+    avg = statistics.mean(score[1:])
+
+    cnt = 0
+
+    for _ in (score[1:]):
+        if _ > avg:
+            cnt += 1
```

Comment:
의미 있는 변수를 `_`로 사용하는 건 지양합니다 😢 
변수 의미가 최대한 드러나게 이름을 지어보는 연습을 해봐요!!
```suggestion
    for scoreOfOneStudent in (score[1:]):
        if scoreOfOneStudent > avg:
            cnt += 1
```